src/status_im/ui/screens/wallet/views.cljs

```diff
@@ -90,8 +90,9 @@
                           [react/refresh-control {:on-refresh #(re-frame/dispatch [:update-wallet])
                                                   :tint-color :white
                                                   :refreshing false}])}
-      [react/view {:style styles/scroll-top}] ;; Hack to allow different colors for top / bottom scroll view]
       [total-section portfolio-value currency]
       [list/action-list actions
        {:container-style styles/action-section}]
-      [asset-section assets currency]]]))
+      [asset-section assets currency]
+      [react/view {:style styles/scroll-bottom}
+       #_(Hack to allow different colors for top / bottom scroll view)]]]))
```

Comment:
Please use the regular syntax for comments, e.g. two semicolons, etc. 
I get it that you formatted it like this because of the comment, it's perfectly fine if the comment is after a bunch of closed outer parens, as long as it's in the same line it will be obvious to what it refers, e.g.

```
[react/view {:style styles/scroll-bottom}]]]))  ;; Hack to allow different colors for top / bottom scroll view
```

Reply:
@goranjovic :ok_hand: fixed now